tests/unit/components/cooperation-section-view/CooperationSectionView.spec.jsx

```diff
@@ -0,0 +1,52 @@
+import { beforeEach } from 'vitest'
```

Comment:
I hope this is redundant here because we configured that `expect`, `it` and so on are autoimported

Reply:
yeap. you are right